compile.go

```diff
@@ -72,6 +72,38 @@ func EvalAST(ctx context.Context, astPkg *ast.Package, opts ...ScopeMutator) ([]
 	return sideEffects, scope, nil
 }
 
+// EvalOptions is like EvalAST, but only evaluates options.
+func EvalOptions(ctx context.Context, astPkg *ast.Package, opts ...ScopeMutator) ([]interpreter.SideEffect, values.Scope, error) {
+	return EvalAST(ctx, options(astPkg), opts...)
+}
+
+func options(astPkg *ast.Package) *ast.Package {
+	trimmed := &ast.Package{
+		BaseNode: astPkg.BaseNode,
+		Path:     astPkg.Path,
+		Package:  astPkg.Package,
+	}
+	for _, f := range astPkg.Files {
+		var body []ast.Statement
+		for _, s := range f.Body {
+			if s.Type() == "OptionStatement" {
```

Comment:
Can you use a type switch on the statement instead of calling `Type()`? That is more consistent with how we check the type of the nodes and it makes it easier to refactor later as the comparison is against a real type instead of a string literal.

Reply:
I'm glad you called this one out, I was going to ask you about this exact thing! Testing the requested changes now.